docs/configure-upstream-collector.md

```diff
@@ -2,8 +2,8 @@
 
 You can build and configure a [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or extend the [OpenTelemetry Collector Contrib ](https://github.com/open-telemetry/opentelemetry-collector-contrib) distribution to collect logs and metrics and send them to Elastic Observability.
 
-For a more seamless experience, use the Elastic Distribution for the OpenTelemetry collector.
-Refer to the [guided onboarding](guided-onboarding.md) docs or the [manual configuration](manual-configuration.md) docs for more on configuring the Elastic OTel collector.
+For a more seamless experience, use the Elastic Distribution for the OpenTelemetry Collector.
+Refer to the [guided onboarding](guided-onboarding.md) docs or the [manual configuration](manual-configuration.md) docs for more on configuring the Elastic Distribution for the OpenTelemetry Collector.
 
 ## Upstream collector configuration examples

```

Comment:
We want to provide users with configurations that they can copy and paste, maybe store them in this repo for their use.

Reply:
Same as the previous comment.

Reply:
> Elastic Distribution for the OpenTelemetry Collector.

of